tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/stack_trace/stack_trace_component.ng.html

```diff
@@ -18,9 +18,21 @@
 <div class="stack-trace-container">
   <div class="stack-trace-header">
     <span class="stack-trace-title">
-      Stack Trace
+      <div class="stack-trace-title-text">
+        Stack Trace
+      </div>
     </span>
 
+    <div
+      class="stick-to-bottommost-frame"
+      *ngIf="stickToBottommostFrameInFocusedFile"
+      title="Sticking to the bottommost frame in the current source file when
+      navigating executions and graph ops. To remove this sticking, click
+      any non-bottommost stack frame."
+    >
+      Sticking to bottommost frame in file
+    </div>
```

Comment:
Introducing the concept of "stickiness" seems like exposing internal complexity to users, which never alters user behavior (even without help text, I imagine users naturally click on a different frame when they have unwanted sticking).

Could we either drop the word "sticking to" from the text content or consider a visual indicator on the callframe element itself (like styling the bottommost callframe differently)?